test/helpers/image_conversion_helper_test.rb

```diff
@@ -27,69 +28,6 @@ class ImageConversionHelperTest < ActionView::TestCase
     assert "jpg", File.extname(source_image_path)
   end
 
-  test "adds padding to a logo for the correct file size" do
-    source_image_path = "./app/assets/images/brave-lion@3x.jpg"
-    mini_magick_image = MiniMagick::Image.open(source_image_path)
-    mini_magick_image.format "jpg"
-    temp_file = Tempfile.new(["brave_lion_copy", ".jpg"])
-
-    mini_magick_image.write(temp_file)
-
-    temp_file_path = resize_to_dimensions_and_convert_to_jpg(
-      source_image_path: temp_file.path,
-      attachment_type: SiteBanner::LOGO,
-      filename: "brave_lion_logo"
-    )
-
-    add_padding_to_image(
-      source_image_path: temp_file_path,
-      attachment_type: SiteBanner::LOGO,
-      quality: ImageConversionHelper::IMAGE_QUALITY
-    )
```

Comment:
Would you re-add a test to make sure file size output matches as expected?

Reply:
Done